frontend/src/components/CoursePage.js

```diff
@@ -24,6 +24,31 @@ const TutorPage = () => {
         }
         console.log(_tutors);
         let tutorList = [];
+        if (_tutors.length === 0){
+            return(
+                <Box sx={{display:"flex", justifyContent:"center"}}>
+                    <Stack>
+                        <Typography
+                        variant="h4"
+                        justifyContent="center">
+                            No Tutors Available
+                        </Typography>
+                        <Link to="/courses" style={{ textDecoration: 'none', color: "blue" }}>
```

Comment:
I think relying on default browser navigation for this is better; it's not really needed. It also pushes a new history element instead of popping which will cause odd behavior with the back button.